src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -350,16 +365,16 @@ public void clickDeleteInstructorLinkAndCancel(int instrNum) {
     }
 
     public WebElement getNameField(int instrNum) {
-        return browser.driver.findElement(By.id("instructorname" + instrNum));
+        return browser.driver.findElement(By.id(Const.ParamsNames.INSTRUCTOR_NAME + instrNum));
```

Comment:
Probably not possible due to taking in a parameter, which cannot (or rather should not) be predicted in advance.